libs/dh/metering-point/feature-child/src/lib/dh-metering-point-child.component.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * @license
+ * Copyright 2021 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  ChangeDetectionStrategy,
+  Component,
+  NgModule,
+  ViewEncapsulation,
+} from '@angular/core';
+
+const selector = 'dh-metering-point-child';
+
+@Component({
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  encapsulation: ViewEncapsulation.None,
+  selector,
+  styles: [
+    `
+      ${selector} {
+        display: block;
+      }
+    `,
+  ],
+  template: ` <h2>Child</h2> `,
+})
+export class DhMeteringPointChildComponent {}
+
+@NgModule({
+  declarations: [DhMeteringPointChildComponent],
+})
+export class DhMeteringPointChildScam {}
```

Comment:
question: I use  `ViewEncapsulation.None` with inline templates and styles for ETT to get bundle size reductions but it introduces some complexity. Do you want to use this style for DH as well?

question: I use inline SCAMs, templates, and styles for ETT to ease refactoring and eventual migration to standalone components. Do you want to use this style for DH as well?

Reply:
I will remove the `ViewEncapsulation.None` but I think DH should be ready for standalone components as well.

Reply:
suggestion: there is talk of an official Angular migration from SCAMs to standalone components. To be most compatible, we should probably keep selector in the regular format, that is inline in the component metdata.

issue: we should add `display: block;` or `display: inline-block;` for all custom elements such as this one.